features/desktop/notifications.feature

```diff
@@ -101,6 +101,26 @@ Feature: Notifications
     Then I should see "mentioned you in the post"
     And I should have 1 email delivery
 
+  Scenario: someone mentioned me in a comment
+    Given a user with email "bob@bob.bob" is connected with "alice@alice.alice"
+    And "alice@alice.alice" has a public post with text "check this out!"
+    And "bob@bob.bob" has commented mentioning "alice@alice.alice" on "check this out!"
+    When I sign in as "alice@alice.alice"
+    And I follow "Notifications" in the header
+    Then the notification dropdown should be visible
+    Then I should see "mentioned you in the comment"
+    And I should have 1 email delivery
+
+  Scenario: I mark a mention in comment notification as read
```

Comment:
> Why name it generic "mark a notification as read" if it tests mark as read for "mention in comment" notification?

Because it doesn't matter. The "mark as read/unread" feature doesn't depend on the notification type. So it is OK to have a scenario that tests it in the frontend, but I don't want cucumber tests for every notification type to test if you can mark it as read. (that is something you can do in unit-test, which are faster).

And if it doesn't depend on the notification type, I think we should use a generic description. And because you don't need "mention in comment" you can also test it with a normal "mention" and use the existing "And Alice has a post mentioning Bob" step for it.